src/init.c

```diff
@@ -696,6 +696,34 @@ static int hyper_kill_container(char *json, int length)
 	return ret;
 }
 
+static int hyper_remove_container(char *json, int length)
+{
+	struct hyper_remover remover;
+	struct hyper_container *c;
+	struct hyper_pod *pod = &global_pod;
+	int ret = -1;
+
+	if (hyper_parse_remove_container(&remover, json, length) < 0) {
+		goto out;
+	}
+
+	c = hyper_find_container(pod, remover.id);
+	if (c == NULL) {
+		fprintf(stderr, "can not find container whose id is %s\n", remover.id);
+		goto out;
+	}
+
+	if (c->exec.exit != 1) {
+		fprintf(stderr, "container %s has not been stopped\n", remover.id);
```

Comment:
this message is't correct in the case removing container immediately after creating it.
we should rename `exit` field to `status`.
status value:
* CREATED
* INITIALIZED
* RUNNING
* STOPPED

and for RUNNING case, pod->remains-- if container has not been stopped